Make_It_Sense/cpx-cap-touch.py

```diff
@@ -0,0 +1,20 @@
+from adafruit_circuitplayground.express import cpx
+
+# NeoPixel blank out
+cpx.pixels.fill( (0, 0, 0) )
+
+# Set sensitivity of all touch pads
+cpx.adjust_touch_threshold(200)
```

Comment:
Were you running into threshold issues or simply wanted to include the threshold change in the event that people need to alter it later?

Reply:
Just to show it can be done

Reply:
Ok!